### ISSUE TITLE
App windows randomly stop appearing

### ISSUE THREAD

**Issue by ourcore** [lwouis/alt-tab-macos#1588]
**Describe the bug**

Windows for various apps, such as Messages and Chrome, will randomly stop appearing even when they are running and not closed or minimised. Clicking on their icon in the dock and bringing them into view will usually re-add their windows to AltTab. However, in the example below, I had to quit Messages before its window re-appeared in AltTab.

**Screenshots / video**

https://youtu.be/7IQybRG9-xs

**Steps to reproduce the bug**

I haven't identified a pattern to reproduce the issue, but it mostly happens with Messages and Chrome.

**Comment by lwouis** [comment]
I think it's either #1324 (is Messages on another Space or fullscreen in your video?) or #1317

**Comment by ourcore** [comment]
No, I would've mentioned that. I don't use Spaces or fullscreen apps. All of my apps take up half the screen, which I've sized using Magnet. Messages was in the same Space and sized the same as the other apps' windows that appeared in AltTab in my screen recording. I looked through the first page of issues, but couldn't find this one. I think it's like https://github.com/lwouis/alt-tab-macos/issues/1317.
